### PR TITLE
 Unify inference scripts (addresses #21) 

### PR DESCRIPTION
**Pull request by Ryu1845**
Unify inference scripts into a single CLI.

### CODE REVIEW

**Comment by Ryu1845** [comment]
Finished converting tortoise-tts.py. 
Needs to handle the other scripts now.

**Comment by 152334H** [comment]
My apologies for adding the `--voicefixer` arg in the interim.

**Comment by Ryu1845** [comment]
Thanks for telling me :wink:

**Comment by Ryu1845** [comment]
I'll try to finish tomorrow. I can't promise anything though.

**Comment by 152334H** [comment]
sure no worries

**Comment by Ryu1845** [comment]
Only need to add voicefixer and it's done I think.

**Comment by Ryu1845** [comment]
Nvm I need to merge app.py and change the readme too

**Comment by 152334H** [comment]
🥹

**Comment by Ryu1845** [comment]
For some reason the webui is bugging when I call it as a function...

**Comment by Ryu1845** [comment]
I'll open for review without the webui option for now, it's not really necessary tbh.

**Comment by Ryu1845** [comment]
Oh wait I forgot to add the voicefixer toggle in the end :laughing:

**Comment by Ryu1845** [comment]
There, done (exclude launching webui from the cli)

**Comment by Ryu1845** [comment]
@152334H Could you review?

File: scripts/tortoise_tts.py [hunk 1]
```diff
@@ -1,422 +1,362 @@
...
+                if args.output.output_dir:
+                    filename = f"{clip_name}_{candidate_idx:02d}.wav"
+                    save_gen_with_voicefix(audio, os.path.join(args.output.output_dir, filename), squeeze=False, voicefixer=args.general.voicefixer)
+
+        audio = torch.cat(audio_parts, dim=-1)
+        if args.output.output_dir:
+            filename = f'{"-".join(voice)}_combined.wav'
+            torchaudio.save(
+                os.path.join(args.output.output_dir, filename), audio, 24000
+            )
```
Comment (152334H) [inline scripts/tortoise_tts.py:348]: I should add explicit documentation for what the expected result wav filepaths are

**Review by 152334H — COMMENTED** [review]
i'll go test

**Comment by 152334H** [comment]
I would say it's a bit unintuitive that the script shows no output at all when no input text is given, and I'm adding a basic print statement to note when text is being read from stdin.

**Comment by xenotropic** [comment]
One thing that seemingly did not make it in this refactor / merge is the ability to control how text is broken up -- read.py would separate on pipe symbols | . That allowed for the user to choose how chunks are broken up rather than necessarily using --text_split , which often cuts in the middle of sentences, leading to choppier outputs (a pause and "sentence start" emphasis mid-sentence). Feature request; thanks for considering.

**Pull request merged**